src/components/table/JadeTable.jsx

```diff
@@ -119,7 +123,7 @@ export class JadeTable extends React.PureComponent {
   };
 
   render() {
-    const { classes, queryResults, columns, rows, polling } = this.props;
+    const { classes, queryResults, columns, rows, polling, delay } = this.props;
```

Comment:
Not specific to this pr, but could we consider a norm on the UI to order these alphabetically as we do with the static propTypes list? I think it would help with readability

Reply:
sure, we can start fixing those from now on

Reply:
i'll see if i can add something in eslint for it!